src/GameFace/Utils/UsersExtractions.cs

```diff
@@ -0,0 +1,91 @@
+﻿using GameFace.Controllers;
+using GameFace.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GameFace.Utils
+{
+    public static class UsersExtractions
+    {
+
+        public static int GetExperienceUserByTask(int user, int task)
+        {
+            using (var database = new GameFaceContext())
+            {
+                int experience = 0;
+                var users = database.Users.Include(x => x.xP).Where(x => x.id == user);
+                foreach (var u in users)
```

Comment:
The foreach is unnecessary since there will always be only 1 users that matches the userId